api/src/test/java/com/epam/pipeline/security/saml/SAMLUserDetailsServiceImplTest.java

```diff
@@ -220,6 +220,32 @@ public void shouldThrowAuthorizationExceptionForBlockedUser() {
         userDetailsService.loadUserBySAML(credential);
     }
 
+    @Test
+    public void testBlockedValueCaseNonSensitiveness() {
```

Comment:
Could you please add a tree separate negative test instead off one?

Reply:
Done